### PR TITLE
Chore/copy zip clean migrate

### PR DESCRIPTION
**Pull request by BrandonOCasey**
Migrates the clean, copy, and zip grunt tasks to npm scripts

After this pr there will probably be 1 more to convert the remainder to shell commands. From there there will be a pr to remove grunt entirely and to fixup the current shell commands in a way that makes sense.

### CODE REVIEW

**Comment by gkatsev** [comment]
Not sure what the changes are but I saw that the zip file created is only 2.5MB compared to currently where it is around 8.5MB.

Also, I think the package-lock is stale and needs to be regenerated.

**Comment by gkatsev** [comment]
Also, are there alternatives to cross-var? Maybe just a standard script? It seems to be depending on a babel at runtime, so, we have two babel versions being installed in our build tree.

**Comment by BrandonOCasey** [comment]
if we don't care about dev support for windows we can drop it, we could also just use a script yeah

**Comment by BrandonOCasey** [comment]
I saw the size change as well, but it appears the compression is just better. From a diff between the old zip and the new one it was the same.

File: package.json [hunk 1]
```diff
@@ -17,6 +17,13 @@
   "homepage": "https://videojs.com",
   "author": "Steve Heffernan",
   "scripts": {
+    "copy-fonts": "shx cp -R node_modules/videojs-font/fonts build/temp/font",
+    "copy-dist": "shx cp -R build/temp/* dist/",
```
[the remark this answers is not in the document]
Comment (BrandonOCasey) [reply package.json]: I think a lot of this needs to be cleaned up, but for now I feel better just making it do exactly what it did

[the remark this answers is not in the document]
Comment (gkatsev) [reply package.json]: Makes sense, we can fix it up later.

**Comment by gkatsev** [comment]
Yeah, let's just leave it in for now but would be nice to either fix cross-var or do something else long term. Installing two babels isn't great.

Ah, interesting, didn't think about compression for the zip. `unzip ls` seems to confirm the same files exist.

**Comment by gkatsev** [comment]
Ok, confirmed that video.js inside the zip is equivalent to the file outside the zip. Must be different compression settings then.

**Pull request merged**